Mathematics/.texpadtmp/manual.aux

```diff
@@ -0,0 +1,35 @@
+\relax 
```

Comment:
Remove these private configuration files. Put this and other files into .gitignore to not accidentally submit them again